Patch by Robert Scheck <robert@fedoraproject.org> for bitlbee <= 3.2.2, that is
removing the check if the user is root as this causes failures when building RPM
packages. See also http://bugs.bitlbee.org/bitlbee/ticket/1162 at upstream trac.

--- bitlbee-3.2.2/Makefile			2014-07-05 23:40:53.000000000 +0200
+++ bitlbee-3.2.2/Makefile.no-root		2014-07-06 15:50:18.000000000 +0200
@@ -148,14 +148,10 @@
 
 install-systemd:
 ifdef SYSTEMDSYSTEMUNITDIR
-ifeq ($(shell id -u),0)
 	mkdir -p $(DESTDIR)$(SYSTEMDSYSTEMUNITDIR)
 	$(INSTALL) -m 0644 init/bitlbee.service $(DESTDIR)$(SYSTEMDSYSTEMUNITDIR)
 	$(INSTALL) -m 0644 init/bitlbee@.service $(DESTDIR)$(SYSTEMDSYSTEMUNITDIR)
 	$(INSTALL) -m 0644 $(_SRCDIR_)init/bitlbee.socket $(DESTDIR)$(SYSTEMDSYSTEMUNITDIR)
-else
-	@echo Not root, so not installing systemd files.
-endif
 endif
 
 tar:
